documentation/jetty-documentation/src/main/asciidoc/operations-guide/jaspi/chapter.adoc

```diff
@@ -27,29 +27,31 @@ Only modules conforming to the "Servlet Container Profile" with the ServerAuthMo
 Enable the `jaspi` module:
 
 ----
-include::{JETTY_HOME}/modules/jaspi.mod[]
+include::{JETTY_HOME}/modules/ee10-jaspi.mod[]
 ----
 
 [[og-jaspi-xml]]
 ===== Configure JASPI
 
-To enable the `jaspi` module you can use the following command (issued from within the `$JETTY_BASE` directory):
+Activate the `ee{9,10}-jaspi` module that matches your EE platform version.
```

Comment:
If a module exists for ee9, then it should also exist for ee8, since we mechanically generate ee8 from ee9.
I'd say to always use `ee{8,9,10}` everywhere so here `ee{9,10}` -> `ee{8,9,10}`.

Reply:
@olamy do you know whether there is a `ee8-jaspi` module?